js/apps/account-ui/src/utils/useFormatTime.ts

```diff
@@ -0,0 +1,19 @@
+const TIME_FORMAT: Intl.DateTimeFormatOptions = {
+  year: "numeric",
+  month: "long",
+  day: "numeric",
+  hour: "numeric",
+  minute: "numeric",
+};
+
+//todo use user local
+export default function useFormatTime() {
+  return {
+    formatTime: function (
+      time: number,
+      options: Intl.DateTimeFormatOptions | undefined = TIME_FORMAT,
```

Comment:
Same here, no need to specify the `undefined` type.
```suggestion
      options: Intl.DateTimeFormatOptions = TIME_FORMAT,
```